monkey/monkey_island/cc/ui/src/components/pages/RunMonkeyPage/RunOnIslandButton.js

```diff
@@ -23,6 +24,7 @@ class RunOnIslandButton extends AuthComponent {
   constructor(props) {
     super(props);
     this.state = {
+      // TODO: Can't we just use true/false for this?
```

Comment:
There's running, not running, and also a failed state, AFAIR.

Reply:
Yeah... Do we really need all of those on this page?

Reply:
I'd say yes. There's an icon for "running" but no icon for "not running". Without "failed" and a separate icon for it, in case the agent isn't able to start, you don't know whether the problem is with the agent binaries or the server request or something else.

Reply:
:shrug: maybe. Or maybe there's a better way. A red icon doesn't help you distinguish if the problem is with the binary or the server request or something else.